test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperScopeManagerTest.cs

```diff
@@ -24,10 +24,11 @@ public void Begin_CreatesContextWithAppropriateTagName()
 
             // Act
             var executionContext = scopeManager.Begin("p",
-                                                      string.Empty,
-                                                      DefaultExecuteChildContentAsync,
-                                                      DefaultStartWritingScope,
-                                                      DefaultEndWritingScope);
+                selfClosing: false,
+                uniqueId: string.Empty,
+                executeChildContentAsync: DefaultExecuteChildContentAsync,
+                startWritingScope: DefaultStartWritingScope,
+                endWritingScope: DefaultEndWritingScope);
```

Comment:
like the new indentation but please wrap the first argument as well (throughout this class)